themes/default-bootstrap/js/scenes.js

```diff
@@ -82,7 +82,7 @@ $(document).ready(function(){
 
 	$(document).on('click', '.scene_thumb', function(e){
 		e.preventDefault();
-		loadScene($(this).date('id_scene'));
+		loadScene($(this).data('id_scene'));
```

Comment:
Hi, 

you are right, but it will get faster data with : $(this).attr('data-id_scene'); as data('id_scene') is only set by the template and not in js. We can get value of this by "attr" without any risk of any post change in javascript.

Best regards.